### PR TITLE
Minor fixes on Query

### PR DESCRIPTION
**Pull request by karesti**
https://issues.redhat.com/browse/ISPN-13346
https://issues.redhat.com/browse/ISPN-12308

### CODE REVIEW

**Comment by karesti** [comment]
@oraNod if you need to review the wording on tooltips

@diegolovison the stats panel won't show ns or anything like that, we will do it like in global stats, where the tooltip states that values are milliseconds. query are nanoseconds, according to what @gustavonalle told me

File: src/app/Caches/CacheMetrics.tsx [hunk 1]
```diff
@@ -49,7 +50,10 @@ const CacheMetrics = (props: { cacheName: string; display: boolean }) => {
 
     return (
       <Card>
-        <CardTitle>Operation performance values</CardTitle>
+        <CardTitle> <CustomCardTitle
+          title={'Operation performance values'}
+          toolTip={'Average values, in milliseconds, for operations on this cache.'}
```
Comment (oraNod) [inline src/app/Caches/CacheMetrics.tsx:55]:
```suggestion
          toolTip={'Average time, in milliseconds, for read and write operations on this cache.'}
```

File: src/app/Caches/CacheMetrics.tsx [hunk 2]
```diff
@@ -49,7 +50,10 @@ const CacheMetrics = (props: { cacheName: string; display: boolean }) => {
 
     return (
       <Card>
-        <CardTitle>Operation performance values</CardTitle>
+        <CardTitle> <CustomCardTitle
+          title={'Operation performance values'}
```
Comment (oraNod) [inline src/app/Caches/CacheMetrics.tsx:54]:
```suggestion
          title={'Performance of cache operations'}
```

File: src/app/Caches/Query/QueryMetrics.tsx [hunk 3]
```diff
@@ -148,7 +149,12 @@ const QueryMetrics = (props: {
     <Card>
       <CardTitle>
         <Level id={'query-stats'}>
-          <LevelItem>Query stats</LevelItem>
+          <LevelItem>
+            <CustomCardTitle
+              title={'Query stats'}
+              toolTip={'Average values, in nanoseconds, for queries on this cache.'}
```
Comment (oraNod) [inline src/app/Caches/Query/QueryMetrics.tsx:155]:
```suggestion
              title={'Query statistics'}
              toolTip={'Average time, in nanoseconds, for queries on this cache.'}
```

**Pull request merged**